<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formulario</class>
 <widget class="QWidget" name="Formulario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>441</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Apellido:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="4">
     <widget class="QPushButton" name="BotonCancel">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Altura:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Edad:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>DNI:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Peso:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="5">
     <widget class="QPlainTextEdit" name="TextObservaciones">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="9" column="3">
     <widget class="QPushButton" name="BotonOk">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="4">
     <widget class="QLineEdit" name="lineEdit_DNI"/>
    </item>
    <item row="2" column="1" colspan="4">
     <widget class="QLineEdit" name="lineEdit_Nombre"/>
    </item>
    <item row="0" column="1" colspan="4">
     <widget class="QLineEdit" name="lineEdit_Apellido"/>
    </item>
    <item row="4" column="1" colspan="4">
     <widget class="QSpinBox" name="spinBox_Edad"/>
    </item>
    <item row="5" column="1" colspan="4">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_Peso"/>
    </item>
    <item row="6" column="1" colspan="4">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_Altura"/>
    </item>
    <item row="7" column="0" colspan="5">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Observaciones</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
